tests/integration/crunchers/test_cruncher_constant_ratio.py

```diff
@@ -125,6 +128,11 @@ def test_relationship_usage(self, test_downscale_df, add_col):
         test_downscale_df.append(res, inplace=True)
         assert test_downscale_df.filter(variable=follow).equals(res)
 
+        if add_col:
+            assert (test_downscale_df.filter(variable=follow)[add_col] == add_col_val).all()
```

Comment:
That’s true. I guess I don’t trust append because it does stuff like dropping Nan’s unless everything lines up perfectly. So I think we need to either a) test thoroughly here b) add more tests to pyam to make sure we’re happy or c) discourage appending without user checks